### PR TITLE
Don't use all users by default.

### PR DESCRIPTION
**Pull request by ocefpaf**
Installing for all users on Windows with admin powers can be disastrous. So we should not recommend it as the `default` option.

This PR is not ready yet.

Ping @rsignell-usgs

### CODE REVIEW

File: webpage/other_resources.md [hunk 1]
```diff
@@ -15,21 +15,20 @@ you can install it by typing `conda install anaconda` using miniconda.
 ## Install
 
 Download and install the appropriate Miniconda installer from [http://conda.pydata.org/miniconda.html](http://conda.pydata.org/miniconda.html).
-With Anaconda you can create environments that use any Python version (e.g. Python 2.7 or Python 3.5).
-You should install the latest Python 3.5,
+With Anaconda you can create environments that use any Python version (e.g. Python 2.7 or Python 3.x).
+You should install the latest Python 3.x,
```
Comment (ocefpaf) [inline webpage/other_resources.md]: With `3.6` out I guess it is easier to use `3.x`.

**Comment by ocefpaf** [comment]
@rsignell-usgs aside 1 we have:

 **ASIDE 1:** If you are using ArcGIS,
  or any other Python distribution,
 you should **uncheck** the boxes in the install script so that Anaconda is **not** made your default Python,
 nor added to your path.
 You then need to activate `conda` by opening a command prompt terminal and typing:
 
 ```
 C:\Miniconda3\Scripts\activate
 conda install console_shortcut
 ```

Can you check if installing `console_shortcut` is still needed? Someone once told me that `console_shortcut` would be included in latest `miniconda`.

**Comment by ocefpaf** [comment]
Also, in the section

---

### If Miniconda is not your default python environment...

If you choose not to add Miniconda Python Distribution to your `~.bashrc` or Windows path,
you must remember to activate the IOOS environment every time,
by typing in a command prompt

```
C:\Miniconda3\Scripts\activate IOOS3  # Windows
export PATH=$HOME/miniconda3/bin:$PATH && source activate IOOS3  # OSX and Linux
```

---

We probably need to change the Windows instructions because the path will be `C:\<someuserspace>Miniconda3`. I am not a Windows expert but I do know that there are some env vars for that. Do you know which one?

File: webpage/other_resources.md [hunk 2]
```diff
@@ -15,21 +15,20 @@ you can install it by typing `conda install anaconda` using miniconda.
...
 and if find out later you need a Python 2.7 environment,
 you can create one.
 Windows users also need to choose between 32-bit (old Windows XP) or 64-bit (modern Windows) versions.
 
 ### Windows
 
 Run the installer.
-Choose *For all users* instead *For just myself* to `miniconda` is installed outside
-your user `namespace`. Also, make sure both boxes that sets Miniconda as your default Python,
+Choose *Just myself* instead *All users* so `miniconda` is installed in your user `namespace`.
```
Comment (rsignell-usgs) [inline webpage/other_resources.md]: "Just me", not "just myself"

Comment (rsignell-usgs) [inline webpage/other_resources.md]: Acutally "Just Me"

Comment (rsignell-usgs) [inline webpage/other_resources.md]: "Just Me" instead of "All Users"

File: webpage/other_resources.md (hunk 1 above)
Comment (rsignell-usgs) [inline webpage/other_resources.md]: agree

**Comment by rsignell-usgs** [comment]
I do not check the boxes when installing miniconda.
I then launch the Anaconda Command Prompt, which has miniconda in the path.   Then I type "activate IOOS3".    

My colleague here however loves the "Anaconda Navigator" GUI, however.  "she types "anaconda-navigator".  

And yes, you don't need to install the console on windows anymore -- it's part of minconda for windows.

**Pull request merged**